recipes/libcufile/meta.yaml

```diff
@@ -0,0 +1,160 @@
+{% set target_name = "sbsa-linux" %}  # [aarch64]
+{% set extension = "tar.xz" %}  # [not win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/{{ name }}/{{ platform }}/{{ name }}-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: 222c49b29afb544c69542c4e658b0a1bd01fd2306f2136ad6990ccfff482a097  # [linux64]
+  sha256: 4e8d2b525322be9ab6ae574bd1d0325a4bf9c6c3bbdaa1611b5b969b7b34b0c4  # [aarch64]
+  patches:
+    - Makefile.patch
+
+build:
+  number: 0
+  skip: true  # [not linux64]
+
+outputs:
+  - name: libcufile
+    build:
+      missing_dso_whitelist:
+        - "*/libcuda.so.*"                     # [linux]
+      run_exports:
+        - {{ pin_subpackage("libcufile", max_pin="x") }}
+    files:
+      - lib/libcufile*.so.*
+      - etc/cufile.json
+      - man
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - sysroot_{{ target_platform }} 2.17   # [linux]
+      host:
+        - cuda-version {{ cuda_version }}
+      run:
+        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+      run_constrained:
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+    test:
+      commands:
+        - test -f $PREFIX/lib/libcufile.so.*
+        - test -f $PREFIX/lib/libcufile_rdma.so.*
+        - test -f $PREFIX/etc/cufile.json
+    about:
+      home: https://developer.nvidia.com/cuda-toolkit
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_file: LICENSE
+      license_url: https://docs.nvidia.com/cuda/eula/index.html
+      summary: |
+        Library for GPU Direct Storage with CUDA
+      doc_url: https://docs.nvidia.com/gpudirect-storage/api-reference-guide/index.html
+
+  - name: libcufile-dev
+    build:
+      run_exports:
+        - {{ pin_subpackage("libcufile", max_pin="x") }}
+    files:
+      - lib/libcufile*.so
+      - targets/*/lib/libcufile*.so
+      - include/cufile*
+      - targets/*/include/cufile*
+    requirements:
+      host:
+        - cuda-version {{ cuda_version }}
+      run:
+        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+        - libcufile >={{ version }}
+      run_constrained:
+        - libcufile-static >={{ version }}
+    test:
+      commands:
+        - test -f $PREFIX/lib/libcufile.so
+        - test -f $PREFIX/lib/libcufile_rdma.so
+        - test -f $PREFIX/include/cufile*
+    about:
+      home: https://developer.nvidia.com/cuda-toolkit
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_file: LICENSE
+      license_url: https://docs.nvidia.com/cuda/eula/index.html
+      summary: |
+        Library for GPU Direct Storage with CUDA
+      doc_url: https://docs.nvidia.com/gpudirect-storage/api-reference-guide/index.html
+
+  - name: libcufile-static
+    files:
+      - lib/libcufile*_static.a
```

Comment:
We are not shipping static libraries in `$PREFIX/lib` (except for in `cuda-cudart`, iirc). Only in `targets/{{ target_name }}/lib/`.
```suggestion
```